.github/workflows/feature-network-deploy.yml

```diff
@@ -0,0 +1,84 @@
+name: Feature network deploy
+on:
+  workflow_dispatch:
+    inputs:
+      snapshotUrl:
+        description: 'Custom snapshot URL:'
+        required: false
+        default: ""
+      useDefaultSnapshot:
+        description: 'Use default snapshot (0/1):'
+        required: true
+        default: "1"
```

Comment:
Why do we need a separate input for that? If the `snapshotUrl` is an empty string "" it means we should use the default one, no?

Reply:
Done